oracle/build.gradle

```diff
@@ -10,8 +10,18 @@ sourceSets {
     }
 }
 
+cordapp {
+    info {
+        name "CorDapp Option Oracle"
+        vendor "r3"
```

Comment:
I thought we agreed to keep `vendor` set to the "original" value.

Reply:
done